src/deno_dir.rs

```diff
@@ -399,8 +408,17 @@ fn test_src_file_to_url() {
   let (_temp_dir, deno_dir) = test_setup();
   assert_eq!("hello", deno_dir.src_file_to_url("hello"));
   assert_eq!("/hello", deno_dir.src_file_to_url("/hello"));
-  let x = String::from(deno_dir.deps.join("hello/world.txt").to_str().unwrap());
-  assert_eq!("http://hello/world.txt", deno_dir.src_file_to_url(x));
+  let x = String::from(
+    deno_dir
+      .deps
+      .join("hello/world.txt")
+      .to_str()
+      .unwrap(),
+  );
+  assert_eq!(
+    "http://hello/world.txt",
+    deno_dir.src_file_to_url(x)
+  );
```

Comment:
Please don't include unrelated changes. (What version of rustfmt are you using? Mine doesn't do these.)